.github/workflows/nodejs.yml

```diff
@@ -1,4 +1,4 @@
-name: CI for Ambianic UI 
+name: Jest-Cypress Tests
```

Comment:
Is the renaming of this CI script to Jest-Cypress intentional? Is the idea to have separate CI script for release?

Reply:
Agree with the split